docs/hypernode-deploy/getting-started.md

```diff
@@ -12,8 +12,7 @@ myst:
 ---
 caption: Chapters
 maxdepth: 1
+glob:
 ---
-getting-started/install-and-configure-hypernode-deploy.md
```

Comment:
This was manually configured, because the order of the articles make sense. But, perhaps we can prefix them with a number or something.